discord-groups/style.css

```diff
@@ -61,21 +70,31 @@
 .groups {
   overflow: scroll;
 }
+.groups::-webkit-scrollbar {
+  width: 5.5px;
+  height: 5.5px;
+}
+.groups::-webkit-scrollbar-thumb {
+  background: var(--color-buttons-background);
+  border-radius: 20px;
+}
 .groups-list {
   padding: 1rem;
   list-style: none;
-  overflow: scroll;
-  height: 80vh;
+  /* overflow: scroll; */
+  height: 75vh;
 }
 .groups-list li {
   padding: 1rem;
-  border: thin solid black;
+  border: 1px solid rgba(0, 0, 0, 0.2);
+  margin-bottom: 10px;
+  border-radius: 0.5rem;
 }
 .group-name {
   font-size: larger;
 }
 .create-by {
-  opacity: 0.8;
+  opacity: 0.5;
 }
```

Comment:
please try to keep measurements in even numbers and not odd ones because even numbers make it easier for development

Reply:
changed the measurements to even numbers (0.6)